src/models/user.js

```diff
@@ -163,7 +198,7 @@ UserSchema.query.byBody = function(body) {
 
 	return this.where({$or: [
 		{_id: body.user_id},
-		{email: body.email.toLowerCase()}
+		{emails: {"$in": [body.email.toLowerCase()]}}
```

Comment:
Avec MongoDB, tu peux query un array directement, donc la requête originale aurait continué de fonctionner.

```> db.test.insert({email: ["code@max-p.me", "maxime@iptoki.com"]})
WriteResult({ "nInserted" : 1 })
> db.test.find({email: "code@max-p.me"})
{ 
    "_id" : ObjectId("5ea204448d31ee0b3d0f836d"),
    "email" : [ "code@max-p.me", "maxime@iptoki.com" ] 
}
```

En fait, `$in` est pour faire l'opposé: vérifier qu'un champ est dans un array:
```
> db.test.find({email: {$in: ["test@max-p.me", "maxime@iptoki.com"]}})
{
     "_id" : ObjectId("5ea204448d31ee0b3d0f836d"),
    "email" : [ "code@max-p.me", "maxime@iptoki.com" ]
}

> db.test.find({email: {$in: ["test@max-p.me", "bad@max-p.me"]}})
[]
```